tests/phpunit/payment-methods/class-wc-stripe-express-checkout-element-test.php

```
@@ -179,6 +179,130 @@ public function provide_test_add_order_meta() {
+		$subscription = new WC_Subscription();
+		$subscription->set_payment_method( 'stripe' );
+		$subscription->set_payment_method_title( 'Credit Card (Stripe)' );
+		$subscription->update_meta_data( '_wc_stripe_express_checkout_type', $checkout_type );
+		$subscription->save();
+
+		$this->element->maybe_apply_express_title_after_confirmed_intent( $subscription );
+
+		$this->assertSame( $expected, $subscription->get_payment_method_title() );
+		// Meta should be cleaned up after applying.
+		$this->assertSame( '', $subscription->get_meta( '_wc_stripe_express_checkout_type' ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_maybe_apply_express_title_after_confirmed_intent is not named in camelCase. (undefined)

(CamelCaseMethodName)